### PR TITLE
[FLINK-15627] Correct the wrong naming of compareMaps in NFATestUtili…

### PR DESCRIPTION
**Pull request by ayushtkn**

### CODE REVIEW

File: flink-queryable-state/flink-queryable-state-runtime/src/test/java/org/apache/flink/queryablestate/itcases/AbstractQueryableStateTestBase.java [hunk 1]
```diff
@@ -385,7 +385,7 @@ public Integer getKey(Tuple2<Integer, Long> value) {
 			CompletableFuture<JobStatus> jobStatusFuture =
 				clusterClient.getJobStatus(closableJobGraph.getJobId());
 
-			while (deadline.hasTimeLeft() && !jobStatusFuture.get(deadline.timeLeft().toMillis(), TimeUnit.MILLISECONDS).equals(JobStatus.RUNNING)) {
```
Comment (dawidwys) [inline flink-queryable-state/flink-queryable-state-runtime/src/test/java/org/apache/flink/queryablestate/itcases/AbstractQueryableStateTestBase.java]: This is an unrelated change. Could you remove it from the PR?

**Comment by ayushtkn** [comment]
Thanx @dawidwys for the review
Have updated as per comments.
Please give a check once more.

**Comment by dawidwys** [comment]
Thank you for the update. Looks good now. I will merge once we get a green build.

@flinkbot run travis
@flinkbot run azure

**Comment by ayushtkn** [comment]
Thanx @dawidwys for the quick review.
The build seems happy :smile:
